src/ScanItems.js

```diff
@@ -224,17 +233,21 @@ class ScanItems extends React.Component {
   }
 
   checkout = (barcode) => {
-    const { mutator: { checkout } } = this.props;
+    const { mutator: { checkout }, patronBlockOverriddenInfo } = this.props;
     const checkoutData = {
       ...this.getRequestData(barcode),
       loanDate: moment().utc().format(),
     };
 
+    if (!isEmpty(patronBlockOverriddenInfo)) {
+      checkoutData.overrideBlocks = { ...patronBlockOverriddenInfo };
+    }
+
     return this.performAction(checkout, checkoutData);
   }
 
   override = (data) => {
-    const { mutator: { checkout } } = this.props;
+    const { mutator: { checkout }, patronBlockOverriddenInfo } = this.props;
```

Comment:
Could you please split props destructuring to several line?

Reply:
Done